bugtool/cmd/root.go

```diff
@@ -16,13 +16,17 @@ package cmd
 
 import (
 	"context"
+	"crypto/md5"
```

Comment:
Can we use something more up-to-date than md5? While the keys will be hashed and we never manipulate them, let's avoid using obsolete hash algorithms. I guess that if we make it “easy” to find collisions on the hash, and the hash is caught by an attacker, it might make it easier to find potential keys to try on the original network? Sha256 or sha512 would likely be better.

Reply:
Yes noted and absolutely true. md5 was just for testing, I will update it to use modern hashing algorithms.